catalog/nsili/catalog-nsili-common/src/test/java/org/codice/alliance/nsili/common/TestResultDAGConverter.java

```diff
@@ -130,8 +122,99 @@ public void testMandatoryAttributesSuccess() throws Exception {
         assertThat(dag, notNullValue());
     }
 
+    @Test
+    public void testCreateAttribute() throws Exception {
+        MetacardImpl card = getTestCard();
+        Date testDate = new Date(1000);
+        card.setCreatedDate(testDate);
+        card.setModifiedDate(testDate);
+
+        ResultImpl result = new ResultImpl();
+        result.setMetacard(card);
+
+        DAG dag = ResultDAGConverter.convertResult(result,
+                orb,
+                rootPOA,
+                new ArrayList<>(),
+                new HashMap<>());
+        assertThat(dag, notNullValue());
+
+        String attributeName = NsiliConstants.NSIL_PRODUCT + ":" + NsiliConstants.NSIL_CARD + "."
+                + NsiliConstants.STATUS;
+
+        String value = ResultDAGConverter.getAttributeMap(dag)
+                .get(attributeName);
+        assertThat(value, is(NsiliCardStatus.NEW.name()));
+    }
+
+    @Test
+    public void testChangeAttribute() throws Exception {
+        MetacardImpl card = getTestCard();
+        Date testCreateDate = new Date(1000);
+        Date testModifiedDate = new Date(2000);
+        card.setCreatedDate(testCreateDate);
+        card.setModifiedDate(testModifiedDate);
+
+        ResultImpl result = new ResultImpl();
+        result.setMetacard(card);
+
+        DAG dag = ResultDAGConverter.convertResult(result,
+                orb,
+                rootPOA,
+                new ArrayList<>(),
+                new HashMap<>());
+        assertThat(dag, notNullValue());
+
+        String attributeName = NsiliConstants.NSIL_PRODUCT + ":" + NsiliConstants.NSIL_CARD + "."
```

Comment:
I added the testCreateDate and modified date to the getTestCard() method. 

The result is constructed as some of the test methods modify the test card (e.g. modified date is different than created). The attributeName is also different between test cases. The DAG is also parsed based on specific result and attributeName, so that will be different in different test cases.